api/modules/city/serializers.py

```diff
@@ -51,3 +51,14 @@ class CityFactSerializer(serializers.ModelSerializer):
     class Meta:
         model = CityFact
         fields = ('id', 'city_id', 'fact', 'source_text', 'source_url')
+
+
+class CityVisitSerializer(serializers.ModelSerializer):
+    total_trips = serializers.SerializerMethodField()
+
+    class Meta:
+        model = CityVisitLog
+        fields = ['city', 'total_trips']
+
+    def get_total_trips(self, obj):
+        return obj.logs.total_trips
```

Comment:
You can simple get the count of trips here itself instead of storing it as a new field in table